src/sentry/ingest/userreport.py

```diff
@@ -79,6 +80,9 @@ def save_userreport(project, report, start_time=None):
         project=report_instance.project, group=report_instance.group, sender=save_userreport
     )
 
+    if not event:
+        update_user_report.apply_async(kwargs={"user_report": report_instance})
```

Comment:
Could we pass the ids into the task, serializing django models can make upgrading in the future harder.

Reply:
Should we add a minimum delay/wait here? If the queues are empty this task could be executed almost immediately and still not find the event.